shared-bindings/is31fl3741/IS31FL3741.c

```diff
@@ -32,82 +32,35 @@
 #include "shared-bindings/is31fl3741/IS31FL3741.h"
 #include "shared-bindings/util.h"
 #include "shared-module/displayio/__init__.h"
-#include "shared-module/framebufferio/__init__.h"
-#include "shared-module/framebufferio/FramebufferDisplay.h"
 #include "shared-bindings/busio/I2C.h"
 
 //| class IS31FL3741:
-//|     """Displays an in-memory framebuffer to a IS31FL3741 drive display."""
+//|     """Driver for a IS31FL3741 device."""
 //|
 
 //|     def __init__(self, *, width: int) -> None:
```

Comment:
Update this too please.

Reply:
I think I'm missing something for this (and the last) comment. The new description is what the class does or were you looking for something else? (the FrameBuffer wording was awkward so I updated it)

Reply:
This function signature needs all of the arguments it takes below and documentation for each one.

Reply:
Sorry and thanks. Totally was blanking out on what was missing last night.

Reply:
No problem. :-)